content/en/dashboards/querying/_index.md

```diff
@@ -174,6 +174,16 @@ status:error / status:info
 
 **Note**: Formulas are not lettered. Arithmetic cannot be done between formulas.
 
+Here is an example using the `max` operator to find the maximum value between `status:error` and `status:ok`.
+
+```text
+max(status:error, status:ok)
+```
+
+{{< img src="dashboards/querying/arithmetic_7.png" alt="Formula example - max apm" style="width:75%;" >}}
```

Comment:
same comment - both metric queries or metrics / another data source. 

Make sure the example/screenshot you choose is an actual query that users would want to graph